components/hooks/use-bind-key/use-bind-key.tsx

```diff
@@ -0,0 +1,24 @@
+import { useEffect, useRef } from 'react';
+import Mousetrap from 'mousetrap';
+
+/** binds a global key to a handler */
+export function useKeyBind(
```

Comment:
why the file name is use bind key and the function is use key bind. decide if the key or the bind is first

Reply:
done